CHANGES

```diff
@@ -1,6 +1,111 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
 
+================================================================================
+What's new in Charm++ 6.11.0
+================================================================================
+
+This is a feature release, with the following major changes:
+
+Misc:
+
+- Charm++ now builds with CMake by default when using the `./build`
+  script (requires version 3.4 or higher). The old build system is
+  still available by using `./buildold`. Please see
+  https://charm.readthedocs.io/en/latest/charm++/manual.html#installation-with-cmake
+  for more information.
+
+- We have adopted GitHub discussions
+  (https://github.com/UIUC-PPL/charm/discussions) as the preferred
+  venue for Charm++ questions and discussions instead of the
+  charm@cs.illinois.edu mailing list.
+
+- The BigSim emulation facility has been removed from the runtime
+  system.
+
+Charm++ Features & Fixes:
+
+- A significant overhaul of the load balancing infrastructure and the
+  addition of TreeLB, a new, more flexible and extensible base LB,
+  intended to replace the previous CentralLB and HybridBaseLB designs
+  (load balancers using these previous designs are still
+  supported). Some included strategies have been rewritten to use
+  TreeLB, unused and extraneous strategies have been removed. Please
+  see
+  https://charm.readthedocs.io/en/latest/charm++/manual.html#load-balancing
+  for more information.
+
+- Support for intra-node GPU messages, allowing data to be sent
+  to/from GPU memory without touching host memory.
```

Comment:
```suggestion
- Documentation for GPU support in Charm++ is now located at 
  https://charm.readthedocs.io/en/latest/charm++/manual.html#gpu-support. 

- Support for persistent intra-node GPU messages, and experimental support for
  regular (nonpersistent) intra-node GPU messages. This feature allows data to be
  sent to/from GPU memory without going through host memory.
```
Persistent intra-node GPU messaging is currently being implemented with high priority here: https://github.com/UIUC-PPL/charm/pull/3101